apis/r/R/roxygen.R

```diff
@@ -0,0 +1,17 @@
+
+#' @export
```

Comment:
Is this a user-facing and exported class, or a roxygen helper construct?

Reply:
Not necessarily user-facing, but exported so that it can be used in `\Sexpr`

`R CMD check` might (should) throw a warning about it, so I will end up writing docs for it marking it with `@keywords internal`

Reply:
I was worried that R CMD check would complain about `:::` being used in `\Sexpr` but I just checked and saw that [{generics} gets away with it](https://github.com/r-lib/generics/blob/main/R/accuracy.R) so I'll remove the exports

Reply:
_Within the package it is being used_ the `:::` is allowed (and I would say recommended).  It's a very good (it less well understood idiom.  I just used it today (on a branch of the same repo).

The only downside is when you copy and paste code to debug, ie outside the package namespace.  But hiding and clean top-level namespaces are worth it so maybe git it a go!

Reply:
I wasn't aware if that included dynamically-generated docs (where `\Sexpr` is being used), but I'm glad to know that it does!

Reply:
I cannot speak to that. But I can assure you code within package `foo` does not need `foo:::myfunc` and can access `myfunc`.  I would think related / generated docs in the same package behave the same way but neither know nor asserted they do.  I would try it.